azext_hanaonazure/_params.py

```diff
@@ -37,3 +37,22 @@ def load_arguments(self, _):
                    '--hana-db-username'], help="Username for the HANA database to login to for monitoring")
         c.argument('hana_db_password', options_list=[
                    '--hana-db-password', '--hdbpw'], help="Password for the HANA database to login for monitoring")
+    with self.argument_context('sapmonitor') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+        c.argument('monitor_name', options_list=[
+            '--monitor-name', '-n'], help="The name of the SAP monitor", id_part='name')
+    with self.argument_context('sapmonitor create') as c:
```

Comment:
No support for the other subcommands? Or those just don't have arguments?

Reply:
they inherit from the base command.
just checked, all the new commands have parameter descriptions